system-x/services/cert-manager/src/main/java/software/tnb/certmanager/resource/opesnhift/OpenshiftCertManager.java

```diff
@@ -0,0 +1,124 @@
+package software.tnb.certmanager.resource.opesnhift;
```

Comment:
`opesnhift` package

Reply:
fixed in https://github.com/tnb-software/TNB/pull/1291